Gemfile.lock

```diff
@@ -616,65 +618,63 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.11.0)
```

Comment:
### [single_cov](https://github.com/grosser/single_cov)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.3.2 → 1.11.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/single_cov/1.3.2/1.11.0)_]





<details>
<summary>Commits</summary>

A change of **60** commits. See the full changes on [the compare page](https://github.com/grosser/single_cov/compare/v1.3.2...v1.11.0).

These are the first 10 commits:
- (3bfe1e2) [coverage image](https://github.com/grosser/single_cov/commit/3bfe1e24ea12af7aed5b63eb5fe284120d32e731)
- (0beb78c) [fix bugs on newer rubies](https://github.com/grosser/single_cov/commit/0beb78cd92efb37ef096ca5d4bb6c09c36326295)
- (acf06cb) [generate a coverage report](https://github.com/grosser/single_cov/commit/acf06cbf22984ffbfe3c3b009607248cdcdf2941)
- (302171d) [Merge pull request #24 from grosser/grosser/repot](https://github.com/grosser/single_cov/commit/302171dce570561a5b2788d5612413c6b51738e5)
- (d3b5be5) [v1.4.0](https://github.com/grosser/single_cov/commit/d3b5be50864c1611a468ded824cb79df425975b1)
- (0c0b603) [allow forcing line coverage report to support older converag…](https://github.com/grosser/single_cov/commit/0c0b603dc47bef58f597a46ce7f119c0525d94ce)
- (afa2f42) [v1.5.0](https://github.com/grosser/single_cov/commit/afa2f4221e35d53d351e0cf45f6f775269feb275)
- (84137dc) [update and fix singlecov 1.18 breaking dual mode](https://github.com/grosser/single_cov/commit/84137dc74b6591f60d9f77dfe24d7874cd888011)
- (5856be3) [cop](https://github.com/grosser/single_cov/commit/5856be307e83a3a3bc1728726a92e743eaa5ad66)
- (545fabb) [cop](https://github.com/grosser/single_cov/commit/545fabbbb264ec119231310c7701231856cb8323)


</details>